compendium/ConfigurationAdmin/src/ConfigurationPrivate.hpp

```diff
@@ -67,6 +67,13 @@ class ConfigurationPrivate
    * ConfigurationAdminImpl is shutting down or when it is deleting a Configuration.
    */
   virtual void Invalidate() = 0;
+
+  /** Internal method used by {@code ConfigurationAdminImpl} to determine if a configuration
+   * object has been updated. 
+   *
+   * See {@code ConfigurationPrivate#Invalidate}
+   */
+  virtual bool IsUpdated() = 0;
```

Comment:
This should also be updated to whatever you decide to change the name of this function to based on Jeff's suggestion.